mypy/options.py

```diff
@@ -0,0 +1,24 @@
+from mypy import defaults
+from mypy import build
+
+class Options:
+    _instance = None  # type: 'Options'
```

Comment:
Hm. I think all command line flags should be described by `--help`. It's not a promise that they will never change (we'd have to state that somewhere in the formal docs).